CPAC/pipeline/cpac_pipeline.py

```diff
@@ -258,11 +198,16 @@ def run_workflow(sub_dict, c, run, pipeline_timing_info=None, p_name=None,
 
     # TODO ASH Enforce c.run_logging to be boolean
     # TODO ASH Schema validation
+    # change crashfile output to text file, so we can read it without
+    # nipype, or loading in to python, ....
     config.update_config({
         'logging': {
             'log_directory': log_dir,
             'log_to_file': bool(getattr(c.pipeline_setup['log_directory'],
                                         'run_logging', True))
+        },
+        'execution': {
+            'crashfile_format': 'txt'
```

Comment:
@ccraddock the base branch here seems [180 commits behind](https://github.com/radiome-lab/C-PAC/compare/feature/text_configfile_format...FCP-INDI:develop_v1.8_convergence). It looks to me like this insertion plus [the comment above it](https://github.com/FCP-INDI/C-PAC/pull/1451/files#diff-4b55c37114baa52bec4a4a7e3eb0997e58201c63c31486e768460e6b17a93525R201-R202) is the whole intended change here? Like https://github.com/FCP-INDI/C-PAC/commit/1263e1c63a69743ad4ee348038263d25f437ac20?

Reply:
OK, i will fix it. I was using testing/develop_v1.8_convergence and not develop_v1.8_convergence. 

I will rebase and submit a new PR